pkgs/applications/audio/musescore/default.nix

```
@@ -2,27 +2,27 @@
 , alsa-lib, freetype, libjack2, lame, libogg, libpulseaudio, libsndfile, libvorbis
 , portaudio, portmidi, qtbase, qtdeclarative, qtgraphicaleffects
 , qtquickcontrols2, qtscript, qtsvg, qttools
-, qtwebengine, qtxmlpatterns
+, qtwebengine, qtxmlpatterns, qtnetworkauth, qtx11extras
 , nixosTests
 }:
 
 mkDerivation rec {
   pname = "musescore";
-  version = "3.6.2";
+  version = "4.0";
 
   src = fetchFromGitHub {
     owner = "musescore";
     repo = "MuseScore";
     rev = "v${version}";
-    sha256 = "sha256-GBGAD/qdOhoNfDzI+O0EiKgeb86GFJxpci35T6tZ+2s=";
+    sha256 = "sha256-mc3FYLrzEyUHNFH/P57dWKKHl2Lxew1AQBu+70zmLJs=";
   };
 
-  patches = [
-    ./remove_qtwebengine_install_hack.patch
-  ];
-
   cmakeFlags = [
+    # Set version string via cmake so `git` won't be required
+    "-DMUSESCORE_VERSION=${version}"
+    "-DMUSESCORE_VERSION_FULL=${version}"
     "-DMUSESCORE_BUILD_CONFIG=release"
```

According to https://github.com/musescore/MuseScore#release-build the setting these days is this instead:
```suggestion
    "-DCMAKE_BUILD_TYPE=Release"
```

---

> According to [musescore/MuseScore#release-build](https://github.com/musescore/MuseScore#release-build) the setting these days is this instead:

Thanks for the suggestion. Nixpkgs' cmake setup hook sets that flag automatically. Hence `MUSESCORE_BUILD_CONFIG` can be removed. I updated the commit.